ssh-gss.h

```diff
@@ -134,6 +134,7 @@ OM_uint32 ssh_gssapi_test_oid_supported(OM_uint32 *, gss_OID, int *);
 
 struct sshbuf;
 int ssh_gssapi_get_buffer_desc(struct sshbuf *, gss_buffer_desc *);
+int sshpkt_get_buffer_desc(struct ssh *, gss_buffer_desc *);
```

Comment:
These function names don't match, and they should.  Maybe rename `ssh_gssapi_get_buffer_desc` to `sshbuf_get_buffer_desc`?  Or rename `ssh_gssapi_get_buffer_desc` to `sshbuf_gssapi_get_buffer_desc` and `sshpkt_get_buffer_desc` to `sshpkt_gssapi_get_buffer_desc`?  Something like that anyway.

Reply:
The function `ssh_gssapi_get_buffer_desc()` is part of the base openssh, so I would like to avoid touching its name and rewriting all the callers. I used this name since it really describes what it does and I wanted to prevent very long names prefixed with `ssh_gssapi`. But in the end, I think using `ssh_gssapi_sshpkt_get_buffer_desc()` will be the cleanest solution, even though it would require breaking the calls to multiple lines.

Reply:
Ah, I missed that that was in baseline.  I don't have strong feelings either way, then; I can see that `sshpkt_get_buffer_desc` does seem better.

Reply:
I already settled down with the longer name. Merging to move along with the patches.